Upstream: yes

From de71c97def75f047ec0dbaabffb8bc698938e84f Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Wed, 30 Oct 2013 19:19:23 +0100
Subject: [PATCH] Make each plugin optional

---
 plugins/CMakeLists.txt | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index e270078..e752ef1 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -1,8 +1,8 @@
-add_subdirectory (gabble)
-add_subdirectory (butterfly)
-add_subdirectory (idle)
-add_subdirectory (haze)
-add_subdirectory (rakia)
-add_subdirectory (salut)
-add_subdirectory (sunshine)
-add_subdirectory (pintxo)
+macro_optional_add_subdirectory (gabble)
+macro_optional_add_subdirectory (butterfly)
+macro_optional_add_subdirectory (idle)
+macro_optional_add_subdirectory (haze)
+macro_optional_add_subdirectory (rakia)
+macro_optional_add_subdirectory (salut)
+macro_optional_add_subdirectory (sunshine)
+macro_optional_add_subdirectory (pintxo)
-- 
1.8.4.2

