commit 289c66f430dfb842afd36191a0ef82a26111c403
Author: Trever Fischer <wm161@wm161.net>
Date:   Fri May 20 09:28:40 2011 -0400

    Patch by Rex Dieter to fix packaging

diff --git a/QtZeitgeist.pc.in b/QtZeitgeist.pc.in
index fa5c053..cdb7635 100644
--- a/QtZeitgeist.pc.in
+++ b/QtZeitgeist.pc.in
@@ -5,7 +5,7 @@ includedir=${CMAKE_INSTALL_PREFIX}/include
 
 Name: QtZeitgeist
 Description: Qt Zeitgeist library
-Version: 0.0.1
+Version: ${CPACK_PACKAGE_VERSION}
 Requires: QtCore QtDBus
 Libs: -L${CMAKE_INSTALL_PREFIX}/lib -lQtZeitgeist
 Cflags: -I${CMAKE_INSTALL_PREFIX}/include/QtZeitgeist -I${CMAKE_INSTALL_PREFIX}/include/
