From d82ad04ab481ed462d13b21aebbbf01ccf20f6ef Mon Sep 17 00:00:00 2001
From: Teo Mrnjavac <teo@getamarok.com>
Date: Sat, 14 Nov 2009 18:00:34 +0100
Subject: [PATCH 2/2] Revert to a default playlist layout on dialog rejection if the former current layout has been deleted.

---
 src/playlist/layouts/PlaylistLayoutEditDialog.cpp |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/src/playlist/layouts/PlaylistLayoutEditDialog.cpp b/src/playlist/layouts/PlaylistLayoutEditDialog.cpp
index 6d65590..9a061f2 100644
--- a/src/playlist/layouts/PlaylistLayoutEditDialog.cpp
+++ b/src/playlist/layouts/PlaylistLayoutEditDialog.cpp
@@ -410,7 +410,10 @@ void PlaylistLayoutEditDialog::reject()     //SLOT
     DEBUG_BLOCK
 
     debug() << "Applying initial layout: " << m_firstActiveLayout;
-    LayoutManager::instance()->setActiveLayout( m_firstActiveLayout );
+    if( layoutListWidget->findItems( m_firstActiveLayout, Qt::MatchExactly ).isEmpty() )
+        LayoutManager::instance()->setActiveLayout( "Default" );
+    else
+        LayoutManager::instance()->setActiveLayout( m_firstActiveLayout );
 
     QDialog::reject();
 }
-- 
1.6.5.2.433.g23cdb

