From ad740032508e9d7e77db9663c2decb3ff07c9637 Mon Sep 17 00:00:00 2001
From: "Wulf C. Krueger" <philantrop@exherbo.org>
Date: Fri, 29 Nov 2013 23:30:15 +0100
Subject: [PATCH] Fix compile error due to undeclared variable

---
 umbrello/dialogs/classifierlistpage.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/umbrello/dialogs/classifierlistpage.cpp b/umbrello/dialogs/classifierlistpage.cpp
index 1cca92c..6c1986e 100644
--- a/umbrello/dialogs/classifierlistpage.cpp
+++ b/umbrello/dialogs/classifierlistpage.cpp
@@ -346,7 +346,7 @@ void ClassifierListPage::slotActivateItem(QListWidgetItem* item)
     }
 
     if (itemIndex >= 0 && (itemIndex < getItemList().size())) {
-        listItem = getItemList().at( itemIndex );
+        UMLClassifierListItem* listItem = getItemList().at( itemIndex );
 
         //now update screen
         m_docTE->setText( listItem->doc() );
-- 
1.8.4.3

