Upstream: yes

From 0ca6c90f4de5b32717eca392622c856e3358c99f Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Tue, 16 Sep 2014 17:30:06 +0200
Subject: [PATCH] Update test for Qt improvement in 5.4

---
 autotests/karchivetest.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/autotests/karchivetest.cpp b/autotests/karchivetest.cpp
index 6b50f32..2170480 100644
--- a/autotests/karchivetest.cpp
+++ b/autotests/karchivetest.cpp
@@ -785,6 +785,9 @@ void KArchiveTest::testCreateZipError()
     // Otherwise we just lose data.
     KZip zip(QDir::currentPath());
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0) // 523440a15f in QSaveFile makes it fail for existing directories, as one would expect.
+    QVERIFY(!zip.open(QIODevice::WriteOnly));
+#else
     QVERIFY(zip.open(QIODevice::WriteOnly));
 
     writeTestFilesToArchive(&zip);
@@ -793,6 +796,7 @@ void KArchiveTest::testCreateZipError()
     QVERIFY(!zip.addLocalFile(QDir::currentPath(), "bogusdir"));
 
     QVERIFY(!zip.close());
+#endif
 }
 
 void KArchiveTest::testReadZipError()
-- 
2.1.0

