From 992a1826f870e8c095414351e2e9dcc044145786 Mon Sep 17 00:00:00 2001
From: Jan Grulich <jgrulich@redhat.com>
Date: Fri, 8 Aug 2014 14:59:32 +0200
Subject: [PATCH] Add missing include to fix build on some systems

BUG:338124
---
 vpn/openvpn/openvpn.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/vpn/openvpn/openvpn.h b/vpn/openvpn/openvpn.h
index 64c30e4..bc3f2fb 100644
--- a/vpn/openvpn/openvpn.h
+++ b/vpn/openvpn/openvpn.h
@@ -26,6 +26,7 @@
 
 #include "vpnuiplugin.h"
 
+#include <QTextStream>
 #include <QVariant>
 
 class OpenVpnUiPlugin : public VpnUiPlugin
-- 
2.0.4

