From 60f24e7ac0e960cd47a0a095313d9fbb0cd7e24c Mon Sep 17 00:00:00 2001
From: Wulf C. Krueger <philantrop@exherbo.org>
Date: Tue, 2 Nov 2010 13:31:18 +0100
Subject: [PATCH] Include a boost header to fix compilation.

---
 kleopatra/crypto/gui/newresultpage.cpp |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/kleopatra/crypto/gui/newresultpage.cpp b/kleopatra/crypto/gui/newresultpage.cpp
index 96d1cb2..9961c70 100644
--- a/kleopatra/crypto/gui/newresultpage.cpp
+++ b/kleopatra/crypto/gui/newresultpage.cpp
@@ -32,6 +32,8 @@
 
 #include <config-kleopatra.h>
 
+#include <boost/mem_fn.hpp>
+
 #include "newresultpage.h"
 
 #include "resultlistwidget.h"
-- 
1.7.2.3

