From a703d96641a6281c4d32697e79eedef1a55cf8f0 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Sat, 30 Aug 2008 07:00:27 +0200
Subject: [PATCH] Ensure that Kontact plugins for app foo aren't build, of app foo isn't build (by using BUILD_foo:BOOL=FALSE).

---
 kontact/plugins/CMakeLists.txt |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/kontact/plugins/CMakeLists.txt b/kontact/plugins/CMakeLists.txt
index a5a4bff..7b6cd73 100644
--- a/kontact/plugins/CMakeLists.txt
+++ b/kontact/plugins/CMakeLists.txt
@@ -2,25 +2,25 @@ remove_definitions(-DKDE_DEFAULT_DEBUG_AREA=5600) #remove kontact debug area
 add_definitions(-DKDE_DEFAULT_DEBUG_AREA=5602)
 
 if(KMAIL_SUPPORTED)
-  add_subdirectory( kaddressbook ) 
+  macro_optional_add_subdirectory( kaddressbook ) 
 endif(KMAIL_SUPPORTED)
 
 if(QGPGME_FOUND)
   if(KMAIL_SUPPORTED)
-    add_subdirectory( kmail ) 
+    macro_optional_add_subdirectory( kmail ) 
   endif(KMAIL_SUPPORTED)
-  add_subdirectory( korganizer )
+  macro_optional_add_subdirectory( korganizer )
   add_subdirectory( summary )
   add_subdirectory( specialdates )
 #re-enable planner for 4.2 if we have people who want to develop it
 #  add_subdirectory( planner )
 endif(QGPGME_FOUND)
-add_subdirectory( knotes ) 
-add_subdirectory( knode ) 
-add_subdirectory( kjots ) 
-add_subdirectory( akregator ) 
+macro_optional_add_subdirectory( knotes ) 
+macro_optional_add_subdirectory( knode ) 
+macro_optional_add_subdirectory( kjots ) 
+macro_optional_add_subdirectory( akregator ) 
 if(KTIMETRACKER_SUPPORTED)
-  add_subdirectory( ktimetracker ) 
+  macro_optional_add_subdirectory( ktimetracker ) 
 endif(KTIMETRACKER_SUPPORTED)
 if(KMOBILETOOLS_SUPPORTED)
   add_subdirectory( kmobiletools ) 
-- 
1.6.0.1

