Upstream: trivial

From 0167629a5aa9851feb6803c29193bf8d349dd4d8 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Wed, 20 Aug 2014 23:16:58 +0200
Subject: [PATCH] CMakeLists.txt: search properly for dependencies

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 257ce3c..6d196ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -44,11 +44,11 @@ find_package(Phonon4Qt5 4.6.60 NO_MODULE)
 include_directories(BEFORE ${PHONON_INCLUDES})
 add_definitions(-DPHONON_LIB_SONAME=\"${PHONON_LIB_SONAME}\")
 
-find_package(KF5 CONFIG REQUIRED
+find_package(KF5 REQUIRED
     KDELibs4Support
     Activities
 )
-find_package(KF5Konq CONFIG REQUIRED)
+find_package(KF5Konq REQUIRED)
 
 
 ## Dependencies
-- 
2.1.0

