From 19070289f6078483d0ecb0517fd01ed752b6623b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bo=20=C3=98rsted=20Andresen?= <zlin@exherbo.org>
Date: Thu, 28 Apr 2011 20:14:57 +0200
Subject: [PATCH] Fix automagic dependency upon Kipi Plugins.

Upstream: No.

---
 ksnapshot/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ksnapshot/CMakeLists.txt b/ksnapshot/CMakeLists.txt
index 3ae28bd..fa5809b 100644
--- a/ksnapshot/CMakeLists.txt
+++ b/ksnapshot/CMakeLists.txt
@@ -1,11 +1,11 @@
 project(ksnapshot)
 
 find_package(KDE4 REQUIRED)
-find_package(Kipi)
 include(KDE4Defaults)
 include(MacroLibrary)
 include(MacroOptionalAddSubdirectory)
 include(MacroBoolTo01)
+macro_optional_find_package(Kipi)
 
 add_definitions (${QT_DEFINITIONS} ${QT_QTDBUS_DEFINITIONS} ${KDE4_DEFINITIONS})
 include_directories (${QDBUS_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR} ${CMAKE_BINARY_DIR} ${KDE4_INCLUDES})
-- 
1.7.4.3

